packages/help-center/src/components/utils.tsx

```diff
@@ -139,3 +139,10 @@ export const matchSupportInteractionId = (
 		return foundMatch;
 	}
 };
+
```

Comment:
I like this way, but are we sure, we don't do anything in the backend?

I mean, this will enable the frontend, but are we using any different treatment in the backend?

Example, like users being assigned in the backend to use a different version (if so, we can do also in Calypso by adding a new version parameter in the sendMessage hook stating a different version when user is assigned to the experiment)

Secondly, this is not using a real A/B experiment, is just balancing users to be using either one experience or the other. Is that intended?